src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/IMetadataStreamProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
```

Comment:
so are we only doing this per-file for the time being?

Reply:
Only for `ApiCompatibility.csproj` which doesn't yet set `<Nullable>enable</Nullable>`. https://github.com/dotnet/sdk/issues/25760 tracks enabling that for the entire project.